UM/Qt/Bindings/Theme.py

```diff
@@ -243,6 +253,16 @@ def load(self, path: str, is_first_call: bool = True) -> None:
                     name = os.path.splitext(icon)[0]
                     self._icons[detail_level][name] = QUrl.fromLocalFile(os.path.join(base_path, icon))
 
+            deprecated_icons_file = os.path.join(iconsdir, "deprecated_icons.json")
+            if os.path.isfile(deprecated_icons_file):
+                try:
+                    with open(deprecated_icons_file, encoding="utf-8") as f:
+                        data = json.load(f)
+                        for icon in data:
+                            self._deprecated_icons[icon] = data[icon]
+                except:
```

Comment:
Probably better to just catch `EnvironmentError` here and `JSONDecodeError`. That way if we make a mistake like a `TypeError` or something, it would crash and tell us about it.

It's also a good idea to log the errors then, if we catch them.